Backported this code change from Python 2.7 to avoid a test failure.

--- Python-2.6.8/Lib/test/test_pep263.py-orig	2012-12-19 13:33:11.257595413 -0800
+++ Python-2.6.8/Lib/test/test_pep263.py	2012-12-19 13:33:17.715857751 -0800
@@ -1,3 +1,5 @@
+# -*- coding: koi8-r -*-
+
 #!/usr/bin/python2.6
 
 import unittest
